sdk/storage/Azure.Storage.Files.Shares/src/ShareFileClient.cs

```diff
@@ -15,6 +15,7 @@
 using Azure.Storage.Sas;
 using Metadata = System.Collections.Generic.IDictionary<string, string>;
 using Azure.Storage.Common;
+using System.Net.Http.Headers;
```

Comment:
Is this added by accident or is it being used in this file?

Reply:
Removed.